tutorials/vr/vr_starter_tutorial/vr_starter_tutorial_part_one.rst

```diff
@@ -0,0 +1,1146 @@
+
+
+Starting VR
+-----------
+
+Now that we have gone over the VR nodes, let's start working on the project. While in ``Game.tscn``, select the ``Game`` node and make a new script called ``Game.gd``.
+In the ``Game.gd`` file, add the following code:
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    extends Spatial
+
+    func _ready():
+        var VR = ARVRServer.find_interface("OpenVR")
+        if VR and VR.initialize():
+            get_viewport().arvr = true
+            get_viewport().hdr = false
+
+            OS.vsync_enabled = false
+            Engine.target_fps = 90
+            # Also, the physics FPS in the project settings is also 90 FPS. This makes the physics
+            # run at the same frame rate as the display, which makes things look smoother in VR!
+
+ .. code-tab:: csharp
+
+    using Godot;
+    using System;
+
+    public class Game : Spatial
+    {
+        public override void _Ready()
+        {
+            var vr = ARVRServer.FindInterface("OpenVR");
+            if (vr != null && vr.Initialize())
+            {
+                GetViewport().Arvr = true;
+                GetViewport().Hdr = false;
+
+                OS.VsyncEnabled = false;
+                Engine.TargetFps = 90;
+                // Also, the physics FPS in the project settings is also 90 FPS. This makes the physics
+                // run at the same frame rate as the display, which makes things look smoother in VR!
+            }
+        }
+    }
+
+Let's go over what this code does.
+
+_________________
+
+In the ``_ready`` function, we first get the OpenVR VR interface using the ``find_interface`` function in the :ref:`ARVRServer <class_ARVRServer>` and assign it to a variable
+called `VR`. If the :ref:`ARVRServer <class_ARVRServer>` finds an interface with the name OpenVR, it will return it, otherwise it will return ``null``.
+
+.. note:: The OpenVR VR interface is not included with Godot by default. You will need to download the OpenVR asset from the
+          `Asset Library <https://godotengine.org/asset-library/asset/150>`_ or `GitHub <https://github.com/GodotVR/godot-openvr-asset>`_.
+
+The code then combines two conditionals, one to check if the `VR` variable is NOT null (``if VR``) and another calls the initialize function, which returns a boolean based on
+whether the OpenVR interface was able to initialize or not. If both of these conditionals return true, then we can turn the main Godot :ref:`Viewport <class_Viewport>` into
+an ARVR viewport.
+
+If the VR interface initialized successfully, we then get the root :ref:`Viewport <class_Viewport>` and set the `arvr` property to ``true``. This will tell Godot to use the initialized
+ARVR interface to drive the :ref:`Viewport <class_Viewport>` display. After setting the ``arvr`` property to ``true``, we set the ``hdr`` property to ``false``. We do this because
```

Comment:
I think it's worth adding a note here that a version that does support HDR is available on the master repo. I'm planning on updating the version in the asset library once Godot 3.2 has been released